concepts/Common/DevExtreme CLI/30 Migrate to Named Configuration Components.md

```diff
@@ -0,0 +1,46 @@
+DevExtreme Angular v24.2 introduced named configuration components. These are strictly typed and specific to each UI component. Strict typing allows code completion and hinting systems to suggest only valid configurations and detect invalid usage. We recommend upgrading to named configuration components to improve your development experience.
+
+The following code snippet demonstrates named and generic configuration components:
+
+    <!-- tab: app.component.html -->
+    <!-- Named configuration component -->
+    <dx-form>
+        <dxi-form-item></dxi-form-item>
+    </dx-form>
+
+    <!-- Generic configuration component -->
+    <dx-form>
+        <dxi-item></dxi-item>
+    </dx-form>
+
+[note] Starting with v25.2, a warning is displayed in the console if you use generic configuration components.
+
+To upgrade your Angular application from older generic configuration components, you can use DevExtreme CLI. The `migrate angular-config-components` command uses our `devextreme-schematics` package to perform the upgrade. Run the command in your project root:
+
+    npm devextreme migrate angular-config-components
+    // ===== or =====
+    npx devextreme-cli migrate angular-config-components
```

Comment:
Using of this command requires using of angular schematics. maybe we should include info that it is possible to install these required packages OR they can be used via npm cache
Please ask Arman if he can provide additional info in this regards.